XCUITests/FxScreenGraph.swift

```diff
@@ -186,3 +186,14 @@ func createScreenGraph(app: XCUIApplication) -> ScreenGraph {
 
     return map
 }
+
+extension Navigator {
+    func openNewURL(urlString: String) {
+        let newTabScreen = map.namedScenes[NewTabScreen]!
+        let app = map.app
+        newTabScreen.addEdge(BrowserTab) { _ in
+            app.textFields["url"].tap()
```

Comment:
@jhugman debugger is telling me that the inside of {} is skipped when I call it from the script- is there a scope issue?  can't seem to figure out why (I'm new to swift as well)